contrib/README.md

```diff
@@ -0,0 +1,3 @@
+# contrib
+
+Contrib scripts are not officially part of arr.ai, and receive no official support. However they provide functionality that may be broadly useful and examples of how to use different features of arr.ai. As such, they are bundled into the arr.ai repository for easy access.
```

Comment:
put this in a few lines instead of one long line.

Reply:
Ugh, I hate maintaining the length of lines of prose. It's a total waste of time. But I'll do it for now for consistency within this repo.